frontend/src/components/ViewTrips/trip-utils.test.js

```diff
@@ -16,56 +29,73 @@ describe('getCleanedTextInput tests', () => {
-    getUserUidArrFromUserEmailArr: (userEmailArr) => {
-          return new Promise(resolve => {
-            resolve(userEmailArr.map(userEmail => '_' + userEmail + '_'));
-          });
-        },
-}));
 describe('getCollaboratorUidArray tests', () => {
   test('No collaborators entered', async () => {
     const expectedUidArr = [`_${mockCurUserEmail}_`];
     // This is the list that is created when there are no collaborators added
     // (automatically one empty string from the constructor created ref).
     const testEmailArr = [''];
 
-    const testUidArr = await getCollaboratorUidArray(testEmailArr);
+    const testUidArr = await TripUtils.getCollaboratorUidArray(testEmailArr);
 
     expect(testUidArr).toEqual(expectedUidArr);
   });
 
   test('Some added collaborators', async () => {
-    const person1Email = 'p1@gmail.com';
-    const person2Email = 'p2@outlook.com';
     const expectedUidArr = [`_${mockCurUserEmail}_`,
-                            `_${person1Email}_`,
-                            `_${person2Email}_`];
-    const testEmailArr = [person1Email, person2Email];
+                            `_${USER_A_EMAIL}_`,
+                            `_${USER_Z_EMAIL}_`];
+    const testEmailArr = [USER_A_EMAIL, USER_Z_EMAIL];
 
-    const testUidArr = await getCollaboratorUidArray(testEmailArr);
+    const testUidArr = await TripUtils.getCollaboratorUidArray(testEmailArr);
 
     expect(testUidArr).toEqual(expectedUidArr);
   });
 
   test('Some added collaborators and some blank entries', async () => {
-    const person1Email = 'p1@gmail.com';
-    const person2Email = 'p2@outlook.com';
     const expectedUidArr = [`_${mockCurUserEmail}_`,
-                            `_${person1Email}_`,
-                            `_${person2Email}_`];
-    const testEmailArr = ['', person1Email, '', person2Email, ''];
+                            `_${USER_A_EMAIL}_`,
+                            `_${USER_Z_EMAIL}_`];
+    const testEmailArr = ['', USER_A_EMAIL, '', USER_Z_EMAIL, ''];
 
-    const testUidArr = await getCollaboratorUidArray(testEmailArr);
+    const testUidArr = await TripUtils.getCollaboratorUidArray(testEmailArr);
 
     expect(testUidArr).toEqual(expectedUidArr);
   });
 });
+
+describe('moveCurUserEmailToFront tests', () => {
+  test('No users (error is caught in getUserEmailArrFromUserUidArr)', () => {
+    const expectedEmailArr = [mockCurUserEmail];
+    const testEmailInputArr = [];
+
+    const testEmailOutputArr = TripUtils.moveCurUserEmailToFront(testEmailInputArr);
+
+    expect(testEmailOutputArr).toEqual(expectedEmailArr);
+  });
+
+  test('Only the current user', () => {
+    const expectedEmailArr = [mockCurUserEmail];
+    const testEmailInputArr = [mockCurUserEmail];
+
+    const testEmailOutputArr = TripUtils.moveCurUserEmailToFront(testEmailInputArr);
+
+    expect(testEmailOutputArr).toEqual(expectedEmailArr);
+  });
+
+  test('Current user between two others (alphabetical order)', () => {
```

Comment:
You specified alphabetical order here - is there a case where they're not in alphabetical order that you need to test?

Reply:
tldr: no.

See my response to the first comment. I think that will answer your question more fully.